The test suite is able to use the installed hspec-discover, even when
it thinks it is not available.  So comment out the check.

--- a/text-conversions.cabal
+++ b/text-conversions.cabal
@@ -55,7 +55,7 @@ test-suite text-conversions-test-suite
     , text-conversions
     , bytestring
     , hspec
-    , hspec-discover
+    -- , hspec-discover
     , text
   other-modules:
       Data.Text.ConversionsSpec
